scripts/drone/events/release.star

```diff
@@ -59,6 +60,34 @@ def retrieve_npm_packages_step():
         "commands": ["./bin/build artifacts npm retrieve --tag ${DRONE_TAG}"],
     }
 
+def release_pr_step(depends_on = []):
+    return {
+        "name": "create-release-pr",
+        "image": images["curl"],
```

Comment:
Does `curl` not already exist in the default shell in the gh actions env?

Reply:
Crazy that something so core like curl isn't installed in alpine. I suppose that's why it's considered so slim of an OS?

Reply:
yeah `bash` isn't on there either.